### PR TITLE
Documentation of Multispectral support and update install instructions

### PR DESCRIPTION
**Pull request by sandeepgadhwal**
Documentation of Multispectral support and update install instructions

### CODE REVIEW

**Comment by rohitgeo** [comment]
@AngadKundra can you please review?

File: guide/01-getting-started/install-and-set-up.ipynb [hunk 1]
```diff
@@ -184,54 +184,58 @@
    "cell_type": "markdown",
```
Comment (AngadKundra) [inline guide/01-getting-started/install-and-set-up.ipynb:184]: <p>I think these commands will change.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/660/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (sandeepgadhwal) [reply guide/01-getting-started/install-and-set-up.ipynb:184]: @rohitgeo should we use the commands that angad has tested and later on we can update the commands from esri channel.

**Comment by AtmaMani** [comment]
@sandeepgadhwal I made some suggestions, could you review and make updates?

**Comment by sandeepgadhwal** [comment]
<p>changed</p>

---

 <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/660/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by sandeepgadhwal** [comment]
@AtmaMani @AngadKundra I have done the required changes. Thanks for the improvements

**Pull request merged**